deployment_scripts/deploy_linux.sh

```diff
@@ -101,25 +101,25 @@ log_message "Creating island dirs under $ISLAND_PATH"
 mkdir -p "${MONGO_PATH}" || handle_error
 mkdir -p "${ISLAND_BINARIES_PATH}" || handle_error
 
-# Detecting command that calls python 3.7
+# Detecting command that calls python 3.11.2
 python_cmd=""
-if [[ $(python --version 2>&1) == *"Python 3.7"* ]]; then
+if [[ $(python --version 2>&1) == *"Python 3.11.2"* ]]; then
   python_cmd="python"
 fi
-if [[ $(python37 --version 2>&1) == *"Python 3.7"* ]]; then
-  python_cmd="python37"
+if [[ $(python311 --version 2>&1) == *"Python 3.11.2"* ]]; then
+  python_cmd="python311"
 fi
-if [[ $(python3.7 --version 2>&1) == *"Python 3.7"* ]]; then
-  python_cmd="python3.7"
+if [[ $(python3.11 --version 2>&1) == *"Python 3.11.2"* ]]; then
+  python_cmd="python3.11"
 fi
 
 if [[ ${python_cmd} == "" ]]; then
-  log_message "Python 3.7 command not found. Installing python 3.7."
+  log_message "Python 3.11.2 command not found. Installing python 3.11.2."
   sudo add-apt-repository ppa:deadsnakes/ppa
   sudo apt-get update
-  sudo apt-get install -y python3.7 python3.7-dev python3.7-venv
-  log_message "Python 3.7 is now available with command 'python3.7'."
-  python_cmd="python3.7"
+  sudo apt-get install -y python3.11 python3.11-dev python3.11-venv
+  log_message "Python 3.11.2 is now available with command 'python3.11'."
```

Comment:
The command installs the latest python 3.11, not 3.11.2

Reply:
That's what we have in the instructions and there is no actual command for installing 3.11.2. I have tested and it is 3.11.2.

Reply:
It's 3.11.2 because 3.11.2 is the latest 3.11. Once 3.11.3 comes out, this command will install 3.11.3, but the log will say "Python 3.11.2 is now available" which is not true, it's not 3.11.2